vocode/streaming/agent/state_agent.py

```diff
@@ -516,8 +514,6 @@ def from_state(
         agent.chat_history = state.chat_history
         agent.json_transcript = state.json_transcript
         agent.current_state = state.current_state
-        agent.resume_state = state.resume_state
-        agent.resume = lambda _: agent.handle_state(agent.resume_state)
```

Comment:
whoops. We actually were supposed to keep this line, but resume from `current_state`